cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -1076,20 +1082,25 @@ std::pair<rmm::device_uvector<PdaTokenT>, rmm::device_uvector<SymbolOffsetT>> ge
   rmm::cuda_stream_view stream,
   rmm::mr::device_memory_resource* mr)
 {
+  // Transduce() writes symbol offsets that may be as large json_in.size()-1
+  CUDF_EXPECTS(
+    json_in.size() == 0 || (json_in.size() - 1) <= std::numeric_limits<SymbolOffsetT>::max(),
+    "Given JSON input is too large");
```

Comment:
Maybe move this into a function to avoid code duplication

Reply:
Done 👍